sdk/keyvault/keyvault-keys/README.md

```diff
@@ -148,6 +148,29 @@ const url = `https://${vaultName}.vault.azure.net`;
 const client = new KeyClient(url, credential);
 ```
 
+## Specifying the Azure Key Vault service API version
+
+Our Key Vault Keys client currently only supports the Azure Key Vault service versions `7.0` and `7.1-preview`, which we specify as part of the options that can be passed through the constructor. The latest supported API version is `7.1-preview`, which is available as the `LATEST_API_VERSION` export. A use case example follows:
```

Comment:
RBAC and Backup are supported in `7.2-preview`. Did you intend to only mention `7.1-preview`?

Reply:
Thank you Chris!

RBAC and Backup/Restore will be in a separate client. I would like to add 7.2-preview support for keys, secrets and certificates in a separate PR. I believe I need to re-generate the swagger code, is that correct?